data_enrichment.py

```diff
@@ -45,6 +48,14 @@ def format_nom_complet(
         name = f"{prenom if prenom else ''}{formatted_name}"
     if nom_raison_sociale:
         name = nom_raison_sociale
+    if denomination_usuelle_1 or denomination_usuelle_2 or denomination_usuelle_3:
+        all_denomination_usuelle = (
+            f"{denomination_usuelle_1}"
+            f" {denomination_usuelle_2} "
+            f"{denomination_usuelle_3}"
+        )
+        if all_denomination_usuelle:
```

Comment:
same here. It always exists as you declare just above and you already conditionned it.